c/array-tiling/mbpr2_true-unreach-call.c

```diff
@@ -0,0 +1,49 @@
+extern void __VERIFIER_error() __attribute__ ((__noreturn__));
+extern void __VERIFIER_assume(int);
+void __VERIFIER_assert(int cond) { if(!(cond)) { ERROR: __VERIFIER_error(); } }
+
+#define CELLCOUNT
```

Comment:
I guess here you forgot to define the number.

Reply:
(As well as in other versions of this benchmark)

Reply:
Ok. Thanks. I will upload the updated files.